### ISSUE TITLE
Long running test: SendMailAsync_CanBeCanceled_CancellationToken

### ISSUE THREAD

**Issue by karelz** [dotnet/runtime#73447]
Long running test:
- **SendMailAsync_CanBeCanceled_CancellationToken** - `System.Net.Mail.Tests.SmtpClientTest.SendMailAsync_CanBeCanceled_CancellationToken`

Regression on **7/18**
- Test disabled on 8/4 for Mono only in PR #73340
- Test disabled on 8/5 for all Unix platforms in PR #73452

Failures 6/6-8/5 (incl. PRs):
- `x64` is default architecture
- `CoreCLR` is default runtime

Date | Run | Notes
-- | -- | --
9/12 | 2x PR | Alpine.314.Arm64.Open<br/>openSUSE.15.2.Amd64.Open
9/10 | 1x Rolling run | Centos.7.Amd64.Open
~9/10~ | ~1x PR~ | ~Windows.10.Amd64.Server2022.ES.Open~ - Unrelated infra timeout?
9/9 | 2x (1x Rolling run) | **OSX.1200**.Amd64.Open - Mono<br/>Centos.7.Amd64.Open
~9/9~ | ~1x PR~ | ~Windows.10.Amd64.Server2022.ES.Open~ - Unrelated infra timeout?
9/8 | 1x Rolling run | Centos.7.Amd64.Open
~9/8~ | ~1x PR #74623~ | ~Windows.10.Amd64.Server2022.ES.Open~ - Unrelated infra timeout?
9/6 | 1x PR | openSUSE.15.2.Amd64.Open
8/29 | 1x Rolling run | Centos.8.Amd64.Open
~8/5-8/11~ | ~6x PR~ | Only 1 unique PR #73472 - likely specific to the PR, unrelated error
8/5 | 1x PR | Centos 8
8/3 | 1x Rolling run | Debian 10 - Mono
8/2 | 2x PR | Debian 10 - Mono<br/>OpenSUSE 15.2
~8/2~ | ~1x PR~ | ~Windows 8.1~ - Unrelated infra problem
8/1 | 1x PR | Centos 8
7/30 | 1x PR | **OSX 12.00** - Mono
7/29 | 1x PR | Debian 10 - Mono
~7/29~ | ~1x PR~ | ~Windows 10 - Windows Nano 1809 - amd64~ - Unrelated infra timeout?
7/28 | 1x Rolling run | Debian 10 - amd64 - Mono
7/27 | 3x (1x Rolling run) | 2x Centos 8<br/>Debian 10 - Mono
7/25 | 1x Rolling run | Centos 7
7/22 | 2x (1x Rolling run) | 2x openSUSE 15.2
7/21 | 1x PR | Debian 10 - Mono
7/20 | 1x PR | Alpine 314 - **Arm32**
7/18 | 1x Rolling run | Fedora 34

```kusto
let timeouts = (friendlyNamePrefix : string, includePR : bool) {
cluster('engsrvprod.kusto.windows.net').database('engineeringdata').WorkItems
| where Status == "Timeout"
| where FriendlyName startswith friendlyNamePrefix
//| where Finished > datetime('2021-09-07')
| distinct JobId, WorkItemId, Name, FriendlyName, ConsoleUri
| join kind=inner (cluster('engsrvprod.kusto.windows.net').database('engineeringdata').Jobs
    | where ((Branch == 'refs/heads/main') or (Branch == 'refs/heads/master') or (includePR and (Source startswith "pr/")) or (Branch startswith 'refs/heads/release/6.0'))
    | where Type startswith "test/functional/cli/"
        and not(Properties contains "runtime-staging")
    | summarize arg_max(Finished, Properties, Type, Branch, Source, Started, QueueName) by JobId
| project-rename JobType = Type) on JobId
| extend PropertiesJson = parse_json(Properties)
| extend OS = replace_regex(tostring(PropertiesJson.operatingSystem), @'\((.*)\).*|([^\(].*)', @'\1\2')
| extend Runtime = iif(PropertiesJson.runtimeFlavor == "mono", "Mono", iif(PropertiesJson.DefinitionName contains "coreclr", "CoreCLR", ""))
| extend TargetBranch = extractjson("$.['System.PullRequest.TargetBranch']", Properties)
| extend Architecture = PropertiesJson.architecture
| extend Scenario = iif(isempty(PropertiesJson.scenario), "--", PropertiesJson.scenario)
| project-away PropertiesJson
| order by Finished desc
};
timeouts('System.Net.Mail.Functional.Tests', true);
```

**Comment by karelz** [comment]
Test disabled in PR #73452

**Comment by karelz** [comment]
Update: We are re-enabling the test in main via PR #74545 with changes that will hopefully give us more info on the root cause.

**Comment by karelz** [comment]
@wfurt please check Kusto and if there is no new failure in last 5 days (since the test was re-enabled), let's close it as done (for 8.0). Thanks!

**Comment by wfurt** [comment]
Happened once on `main` in last two weeks
https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-heads-main-1ec4a131ebe140708d/System.Net.Mail.Functional.Tests/1/console.bd6b838c.log?helixlogtype=result

**Comment by karelz** [comment]
More hits updated in the top post

**Comment by wfurt** [comment]
@rzikm  can you please check if this was fixed by #76361 or if we still see failures? It is 18 days so it should be probably long enough period to close this if we don't see occurrences.

**Comment by rzikm** [comment]
No hits (of either assert or long running test) in last 2 weeks, so I think it is safe to close this as fixed.
